pkg/apis/externaldns/types.go

```diff
@@ -99,7 +99,7 @@ func (cfg *Config) ParseFlags(args []string) error {
 	// Flags related to providers
 	app.Flag("provider", "The DNS provider where the DNS records will be created (required, options: aws, google, azure, cloudflare, digitalocean, inmemory)").Required().PlaceHolder("provider").EnumVar(&cfg.Provider, "aws", "google", "azure", "cloudflare", "digitalocean", "inmemory")
 	app.Flag("google-project", "When using the Google provider, specify the Google project (required when --provider=google)").Default(defaultConfig.GoogleProject).StringVar(&cfg.GoogleProject)
-	app.Flag("domain-filter", "Limit possible target zones by a domain suffix (optional)").Default(defaultConfig.DomainFilter).StringVar(&cfg.DomainFilter)
+	app.Flag("domain-filter", "Limit possible target zones by a comma separeted list of domain suffixes (optional)").Default(defaultConfig.DomainFilter).StringVar(&cfg.DomainFilter)
```

Comment:
The reason doing this way is, it's much easier to inject multiple domains as a single environmental variable into the pod. But might we work with a variable like this `DOMAINS="--domain-filter foobar.org --domain-filter bla.org"`..